creating_images/index.adoc

```diff
@@ -6,3 +6,7 @@
 :experimental:
 
 This guide provides best practices on writing and testing Docker images that can be used on OpenShift.
+ifdef::openshift-enterprise,openshift-origin[]
```

Comment:
Probably not, I was just trying to think of a reason that this link wouldn't be shown.  If install is keyed to those distros then this is fine.